### DISCUSSION TITLE
PR multiline review

### DISCUSSION THREAD

**Discussion by alexbagirov** [community/community#23002] [New to GitHub]
<p>Hi!</p>
<p>Sorry for probably a silly question, but is there a way to select multiple lines while reviewing a pull request? Currently I can only see a blur plus sign which selects only one line.</p>
<p>Thanks,</p>
<p>Alex</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/alexbagirov">@alexbagirov</a>,</p>
<p>At this time, there is not a way to highlight multiple lines to add a comment on a Pull Request review. However, you can select multiple lines when looking at the file on the branch that the Pull Request is based on and then select <strong>Copy Permalink</strong> which will give you a link to those selected lines as you can see here:</p>
<p><img src="https://user-images.githubusercontent.com/1864667/181074200-97706d2b-c874-4d65-a337-5953838fa278.png" alt="Screen Shot 2018-04-29 at 4.10.39 PM.png" data-base62-sha1="jQygCO4XpSP4u8DJwQOL4BPLCi" width="177" height="166"></p>
<p>Then when you copy that link into a review comment, it will be rendered as those lines of code. That way, you can comment beneath the relevant place in the code but still indicate all of the lines you’re referring to.</p>
<p>Does that help?</p>

**Comment by oleksii-lukin** [comment]
<p>Will this continue working if you delete branch after merge?</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/llamerr">@llamerr</a>,</p>
<p>Yes, the link will still work after the branch is deleted.</p>
<p>Thanks!</p>

**Comment by genuinefafa** [comment]
<p>What about a whole new file? How can I do it there? I cannot see the <code>...</code> thinggie… </p>

**Comment by danielrvt** [comment]
<p>this may be a dumb question but, how do I select multiple lines?</p>

**Comment by connesy** [comment]
<blockquote>
<p><a class="mention" href="/danielrvt">@danielrvt</a> wrote:</p>
</blockquote>
<blockquote>
<p>this may be a dumb question but, how do I select multiple lines?</p>
</blockquote>
<p>When in diff vew, click the ‘…’ in the right hand corner of a file and choose ‘View file’. This will open that file in the new branch. Then click on the first line you want to select, hold shift and click on the last line. Then copy-paste the whole url into your comment on the PR, or press the ‘…’ by the first selected line and choose ‘Copy permalink’.</p>

**Comment by billnguyense** [comment]
<p>Good news: github has just launched the feature. Now you can highlight multiple lines in a pull request diff and add a comment, all at once.</p>
<p><a href="https://twitter.com/github/status/1179101186437324801" rel="nofollow noopener">https://twitter.com/github/status/1179101186437324801</a></p>
